### PR TITLE
Add this stupid ass image as blacklist

### PR DESCRIPTION
**Pull request by NoahDevelops**

### CODE REVIEW

**Comment by FC5570** [comment]
youre using `var`? do you live under a rock?

**Comment by NoahDevelops** [comment]
It is currently 9:40am, I am sitting at my desk in a fucking English class do you really think I have time to think about what declaration to use?

**Comment by NoahDevelops** [comment]
also, *you're

**Comment by FC5570** [comment]
thinking about what declaration to use takes less than a second, you had time to code all of this, didn't you have a second to think what to use?

**Comment by Bocon778** [comment]
beans

**Comment by NoahDevelops** [comment]
What's the difference between var let or whatever the fuck? You are declaring a variable regardless, it doesn't matter how the hell you do it. What are you gonna do? Scream at discord for using var instead of let?

**Comment by Bocon778** [comment]
calm down

**Comment by NoahDevelops** [comment]
Okay Bacon778

**Comment by FlaringPhoenix** [comment]
You should know the difference between var and let lol.

**Comment by Bocon778** [comment]
its bocon bitch

**Comment by NoahDevelops** [comment]
> 
> 
> You should know the difference between var and let lol.

What I mean to say is in that in this instance. Using either var or let wouldn't matter right?

**Comment by FlaringPhoenix** [comment]
> > You should know the difference between var and let lol.
> 
> What I mean to say is in that in this instance. Using either var or let wouldn't matter right?

In this instance, no it doesn't really matter. Just was saying you should know the difference as it's quite interesting.

**Comment by NoahDevelops** [comment]
> 
> 
> > > You should know the difference between var and let lol.
> > 
> > 
> > What I mean to say is in that in this instance. Using either var or let wouldn't matter right?
> 
> In this instance, no it doesn't really matter. Just was saying you should know the difference as it's quite interesting.

I do know the *exact* difference between them if that's what you mean. But in similarity, I'm talking about how they both serve the same purpose

**Comment by NoahDevelops** [comment]
Currently No Longer Needed - Taken care by solo

**Comment by NoahDevelops** [comment]
Reopened request because its happening again for fuck sake

File: bot/discord/events/message.js [hunk 1]
```diff
@@ -37,6 +37,19 @@ module.exports = (client, message) => {
         }
     }
 
+    if(message.content.includes("https://cdn.discordapp.com/attachments/754441222424363088/831566529342078976/image0-3.png")) {
+     
+        var role = message.guild.roles.cache.get("726829710935457872")
+        
+        message.reply("Your a freaking idiot get muted khed")
```
Comment (itzJOHv) [inline bot/discord/events/message.js:44]: *You're

Comment (NoahDevelops) [reply bot/discord/events/message.js:44]: i still don't like this

**Comment by itzJOHv** [comment]
lol

**Comment by NoahDevelops** [comment]
I request commit

**Comment by itzJOHv** [comment]
huh

**Comment by itzJOHv** [comment]
didnt u think that some1 can download this image and link will be different

**Comment by NoahDevelops** [comment]
> 
> 
> didnt u think that some1 can download this image and link will be different

then we shall block that one too

**Comment by itzJOHv** [comment]
> > didnt u think that some1 can download this image and link will be different
> 
> then we shall block that one too

ig better will be reading img content and if it *matches* it'll do this

**Comment by Microwave-6443** [comment]
> Reopened request because its happening again for fuck sake

so what if people like contributing to the growth of a meme. stop getting angry at absolutely nothing... if we block this then we might as well just delete the #memes channel and block every other meme...

**Comment by DarkerInker** [comment]
image

**Comment by danielpmc** [comment]
Permission to talk - Locked

**Pull request closed without merging**